azure_pipelines/msal-release-ado-trigger.yml

```
@@ -6,4 +6,14 @@ trigger:
     include:
     - CHANGELOG.md
 
-pr: none
\ No newline at end of file
+pr: none
+
+jobs:
+- job:
+  displayName: "Trigger MSAL Release - OneBranch Pipeline"

```

nit: would it possible to add the link to the release pipeline job on ADO? Something like:
```suggestion
  displayName: "Trigger MSAL Release - OneBranch Pipeline" # The triggered pipeline job can be found here: https://identitydivision.visualstudio.com/IDDP/_git/MSAL-ObjC-Pipelines?path=/Pipeline%20YAMLs/msal-release-onebranch.yml
```

---

I didn't want to reveal too much info publicly hence defined pipelines variables for the project, organization and pipeline id this would trigger. Reading the pipeline variables, you can navigate to the pipeline being triggered.